### PR TITLE
Update metallb version to 0.14.8

### PR DESCRIPTION
**Pull request by eaudetcobello**
* Bump metallb chart to v0.14.8
* Add convenience update-metallb-chart.py script
* Change image repos to github.com/canonical/metallb-{controller,speaker} 
* Don't bump frr since we don't enable it and don't offer a way to enable it

### CODE REVIEW

File: build-scripts/hack/upstream-images.yaml [hunk 1]
```diff
@@ -49,11 +49,11 @@ sync:
   - source: quay.io/tigera/operator:v1.34.0
     target: ghcr.io/canonical/k8s-snap/tigera/operator:v1.34.0
     type: image
-  - source: quay.io/metallb/controller:v0.14.5
-    target: ghcr.io/canonical/k8s-snap/metallb/controller:v0.14.5
+  - source: quay.io/metallb/controller:v0.14.8
+    target: ghcr.io/canonical/k8s-snap/metallb/controller:v0.14.8
```
Comment (addyess) [inline build-scripts/hack/upstream-images.yaml:53]: wait?  isn't it this:

```suggestion
    target: ghcr.io/canonical/metallb-controller:v0.14.8
```

Comment (eaudetcobello) [reply build-scripts/hack/upstream-images.yaml:53]: Well I think this file is outdated anyway. It used to be that we pulled the images from quay and synced them to ghcr. But now we build the rocks. We don't want quay images to replace the rocks.

**Comment by eaudetcobello** [comment]
Seeing this error:
```
root@ubuntu:~# sudo k8s kubectl logs metallb-controller-55bbfd69f4-vd68v -n metallb-system
cannot run daemon: directory "/var/lib/pebble/default" not writable
```

**Review by bschimke95 — APPROVED** [review]
LGTM

**Pull request merged**